README.md

```diff
@@ -135,17 +147,17 @@ Example: `npx hardhat functions-read --network polygonMumbai --contract 0x787Fe0
+| `functions-request`                | Initiates a request from a _FunctionsConsumer_ client contract using data from the Functions request config file                     | `network`: Name of blockchain network, `contract`: Address of the client contract to call, `subid`: Billing subscription ID used to pay for the request, `gaslimit` (optional): Maximum amount of gas that can be used to call _fulfillRequest_ in the client contract (defaults to 100,000 & must be less than 300,000), `requestgas` (optional): Gas limit for calling the _executeRequest_ function (defaults to 1,500,000), `simulate` (optional): Flag indicating if simulation should be run before making an on-chain request (defaults to true), `configpath` (optional): Path to request config file (defaults to `./Functions-request-config`.js_)                                                              |
+| `functions-read`                   | Reads the latest response (or error) returned to a _FunctionsConsumer_ or _AutomatedFunctionsConsumer_ client contract               | `network`: Name of blockchain network, `contract`: Address of the client contract to read, `configpath` (optional): Path to request config file (defaults to `./Functions-request-config`.js_)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| `functions-deploy-auto-client`     | Deploys the _AutomatedFunctionsConsumer_ contract and sets the Functions request using data from the Functions request config file   | `network`: Name of blockchain network, `subid`: Billing subscription ID used to pay for Functions requests, `gaslimit` (optional): Maximum amount of gas that can be used to call _fulfillRequest_ in the client contract (defaults to 250000),  `verify` (optional): Set to `true` to verify the deployed _AutomatedFunctionsConsumer_ contract (defaults to `false`), `simulate` (optional): Flag indicating if simulation should be run before making an on-chain request (defaults to true), `configpath` (optional): Path to request config file (defaults to `./Functions-request-config`.js_) |
```

Comment:
for now i think we should leave as is and revisit later because it could break some of the docs...this feels like it could break the command for people who've used the repo before.  Not sure if the benefits outweight the disbenefits here on the semantics of it